src/state/tokenPrices/hooks.ts

```diff
@@ -0,0 +1,70 @@
+import { ChainId } from '@kyberswap/ks-sdk-core'
+import { useEffect, useMemo } from 'react'
+
+import { NETWORKS_INFO } from 'constants/networks'
+import { useActiveWeb3React } from 'hooks'
+import { useAppDispatch, useAppSelector } from 'state/hooks'
+import { isAddressString } from 'utils'
+
+import { updatePrices } from '.'
+
+export const useTokenPrices = (addresses: Array<string>) => {
+  const tokenPrices = useAppSelector(state => state.tokenPrices)
+  const dispatch = useAppDispatch()
+  const { chainId } = useActiveWeb3React()
+
+  const addressKeys = addresses
+    .sort()
+    .map(x => x.toLowerCase())
+    .join(',')
+
+  const tokenList = useMemo(() => {
+    return addressKeys.split(',').filter(Boolean)
+  }, [addressKeys])
+
+  const unknownPriceList = useMemo(() => {
+    return tokenList.filter(item => tokenPrices[`${item}_${chainId}`] === undefined)
+  }, [tokenList, chainId, tokenPrices])
+
+  useEffect(() => {
+    const fetchPrices = async () => {
+      const chainIdMapping: Partial<Record<ChainId, string>> = {
+        [ChainId.BSCMAINNET]: 'bsc',
+        [ChainId.BTTC]: 'bttc',
+      }
```

Comment:
either move this into networks config or use from [here](https://github.com/KyberNetwork/kyberswap-interface/blob/improve-farm-load/src/hooks/useAggregatorStats.ts#L7-L12)

Reply:
done

Reply:
you should have moved it to `constants/network.ts`

Reply:
check this. https://github.com/KyberNetwork/kyberswap-interface/blob/main/src/hooks/useAggregatorStats.ts#L7-L8
Danh and Hung discussed about this before

Reply:
I meant [this file](https://github.com/KyberNetwork/kyberswap-interface/blob/main/src/constants/networks.ts), not those config files folder

Reply:
anything related to network should be there, even we dont make it separated field in NETWORK_INFO

Reply:
done